saleae/version.py

```diff
@@ -1 +1 @@
-__version__ = '0.3.0'
+__version__ = '0.4.0'
```

Comment:
Mmm, the version bump was in a different commit but I did not limit the push request commit range, stupid me...